docs/developer/tutorials/build-rocks-in-launchpad.rst

```diff
@@ -0,0 +1,185 @@
+.. _builds-rocks-in-launchpad:
+
+Build Rocks in Launchpad
+========================
+
+Rocks are Ubuntu LTS-based `OCI <https://opencontainers.org/>`_-compliant
+container images designed for security, stability, and reliability in
+cloud-native environments. Read more about rocks `here
+<https://documentation.ubuntu.com/rockcraft/stable/explanation/rocks/>`_.
+
+Building rocks on Launchpad ensures clean, reproducible builds across multiple
```

Comment:
"reproducible" is a fixed term which comes with expectations.. that is you would be always able to build the very same image and receive the very same artifact.. I am not sure whether this is the case, actually, I am pretty sure this is not the case. Imagine you have not pinned one dependency and rebuilt it later on when there is a new version of the dependency which gets pulled in.